packages/lodestar/src/chain/chain.ts

```diff
@@ -356,6 +360,9 @@ export class BeaconChain extends (EventEmitter as { new(): ChainEventEmitter })
         const finalizedBlockRoot = newState.finalizedCheckpoint.root;
         const finalizedBlock = await this.db.block.get(finalizedBlockRoot);
         this.logger.important(`Epoch ${computeEpochOfSlot(this.config, finalizedBlock.slot)} is finalized!`);
+        
+        this.metrics.currentFinalizedEpoch.inc();
```

Comment:
Not sure, but htere could be case if there is not enough attestations to skip finalization. Not sure it's always increment

Reply:
Then this if statement is incorrect

Reply:
yeah but my point it metrics will have inavlid finalized epoch because on next finalization it will increment by +2

Reply:
you can't finalize epoch 3 without having finalized epoch 2? that doesn't make sense?

Reply:
how come? Epoch won't get justified if 2/3 validators don't agree attest and after epoch passes they will all start attesting next epoch, right?

Reply:
i actually don't know...

Reply:
we can use a `.set(newState.finalizedCheckpoint.epoch)` to avoid this